_SUMMARY.md

```diff
@@ -252,6 +252,8 @@
 * [PhpStorm](products/phpstorm/phpstorm.md)
     * [Working with the PHP Open API](products/phpstorm/php_open_api.md)
         * [Breaking Changes](products/phpstorm/php_open_api_breaking_changes.md)
+            * [2020.2](products/phpstorm/php_open_api_breaking_changes_2002.md)
```

Comment:
can we name the page "2020_2" or "202" etc.?

Reply:
I think we can, why not.

Reply:
and put latest on top, thanks